### PR TITLE
Assembler model

### PR DESCRIPTION
**Pull request by nirmal864**
#66 added schema

### CODE REVIEW

File: Server/src/Models/Assembler.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+// Schema
+const assembler_schema = mongoose.Schema({
```
Comment (Pyxsys) [inline Server/src/Models/Assembler.js]: Rename to `schedule_schema`. adjust exports accordingly.

**Review by Pyxsys — CHANGES_REQUESTED** [review]

File: Server/src/Models/Assembler.js (hunk 1 above)
Comment (nirmal864) [reply Server/src/Models/Assembler.js]: changed

**Review by Pyxsys — APPROVED** [review]
#66 is not satisfied with this merge.

**Comment by berkbozok** [comment]
looks good

**Pull request merged**